### PR TITLE
XRT support for SUSE 15

### PR DESCRIPTION
**Pull request by Himanshu-xilinx**
*Steps to build and install XRT on SUSE- https://confluence.xilinx.com/display/~hchoudha/Steps+To+Build+XRT+on+SUSE

### CODE REVIEW

**Comment by chvamshi-xilinx** [comment]
retest this please

File: src/CMake/config/postinst.in [hunk 1]
```diff
@@ -147,6 +147,10 @@ if [ -z $VERSION ] ; then
         pip install --ignore-installed numpy==1.8
     fi
     pip install pyopencl
+    # following change required because of mismatch between python and pip version
+    if [ $FLAVOR = "sles" ] ; then
+        pip2 install pyopencl
+    fi
```
Comment (stsoe) [inline src/CMake/config/postinst.in]: We decided this wasn't needed, right?

**Comment by maxzhen** [comment]
do not merge until a branch is created.

**Comment by salindac** [comment]
retest this please

**Comment by chvamshi-xilinx** [comment]
> do not merge until a branch is created.

Hi @maxzhen , PU2 branch is already created. Is there any other branch planned from master?
If not, we can remove the donot merge tag

**Comment by chvamshi-xilinx** [comment]
retest this please

**Pull request merged**